likecoin/includes/constant/options.php

```diff
@@ -46,6 +46,8 @@
 define( 'LC_MATTERS_INFO', 'lc_matters_info' );
 define( 'LC_ISCN_DEV_INFO', 'lc_iscn_dev_info' );
 define( 'LC_ISCN_INFO', 'lc_iscn_info' );
+define( 'LC_ARWEAVE_INFO', 'lc_arweave_info');
+define( 'LC_IPFS_INFO', 'lc_ipfs_info');
```

Comment:
I think we can store IPFS from arweave into `lc_arweave_info`, then everytime we need to use IPFS hash, make a function that gets both `LC_MATTERS_INFO` and `LC_ARWEAVE_INFO` and take one of the hash as the IPFS hash we use. That way we can store both IPFS hashes without conflict

Reply:
As discussed, will keep both ipfsHash and store them both in content fingerprint when submitting to ISCN.